torch/testing/_internal/common_methods_invocations.py

```diff
@@ -13977,6 +13977,65 @@ def reference_flatten(input, start_dim=0, end_dim=-1):
                 "TestNormalizeOperators",
                 "test_normalize_operator_exhaustive",
             ),
+            DecorateInfo(
+                unittest.expectedFailure,
+                "TestCompositeCompliance",
+                "test_backward",
+                dtypes=[torch.float32],
+                active_if=(TEST_WITH_TORCHINDUCTOR)
+            ),
+            DecorateInfo(
+                unittest.expectedFailure,
+                "TestMathBits",
+                "test_neg_view",
+                dtypes=[torch.float64],
+                device_type='cuda',
+                active_if=(TEST_WITH_TORCHINDUCTOR)
+            ),
+            DecorateInfo(
+                unittest.expectedFailure,
+                "TestCompositeCompliance",
+                "test_forward_ad",
+                dtypes=[torch.float32],
+                active_if=(TEST_WITH_TORCHINDUCTOR)
+            ),
+            DecorateInfo(
+                unittest.expectedFailure,
+                "TestBwdGradients",
+                "test_fn_grad",
+                dtypes=[torch.float64, torch.complex128],
+                device_type='cuda',
+                active_if=(TEST_WITH_TORCHINDUCTOR)
+            ),
+            DecorateInfo(
+                unittest.expectedFailure,
+                "TestBwdGradients",
+                "test_fn_grad",
+                dtypes=[torch.float64],
+                device_type='cpu',
+                active_if=(TEST_WITH_TORCHINDUCTOR)
+            ),
+            DecorateInfo(
+                unittest.expectedFailure,
+                "TestBwdGradients",
+                "test_fn_gradgrad",
+                dtypes=[torch.float64, torch.complex128],
+                active_if=(TEST_WITH_TORCHINDUCTOR)
+            ),
+            DecorateInfo(
+                unittest.expectedFailure,
+                "TestFwdGradients",
+                "test_fn_fwgrad_bwgrad",
+                dtypes=[torch.float64, torch.complex128],
+                active_if=(TEST_WITH_TORCHINDUCTOR)
+            ),
+            DecorateInfo(
+                unittest.expectedFailure,
+                "TestFwdGradients",
+                "test_forward_mode_AD",
+                dtypes=[torch.float64, torch.complex128],
+                active_if=(TEST_WITH_TORCHINDUCTOR)
+            ),
```

Comment:
Where are we tracking these xfails? Why do we expect this PR to cause these xfails?

Reply:
To elaborate, you say you are tracking the xfails at https://github.com/pytorch/pytorch/issues/112424 but the issue doesn't explain why the tests are failing, it seems to only be a list of failures that showed up when you made this change. You state that they are "latent bugs". This is believable, but you haven't presented any *evidence* that these are latent bugs. One easy way to get this evidence is to run the test with TORCH_LOGS=+dynamo before and after your PR and show something, e.g., previously we graph breaked but now we don't (or vice versa).

All of these xfails make me nervous because no one cares if a problem was a latent bug or not: if you change some unrelated codepath so that the latent bug is triggered, it's still a bug, people are still going to have problems with it if you change it this way. In some circumstances, it may still be right to land the change anyway, but you have at least some burden of proof to argue that this is OK.